poc_mobile_verifier/src/cli/server.rs

```diff
@@ -53,14 +53,28 @@ impl Cmd {
         .create()
         .await?;
 
-        // Subnetwork rewards
-        let (subnet_rewards_tx, subnet_rewards_rx) = file_sink::message_channel(50);
+        // Radio share rewards
+        let reward_manifest = Manifest::default();
+        let (radio_rewards_tx, radio_rewards_rx) = file_sink::message_channel(50);
         let mut subnet_rewards = file_sink::FileSinkBuilder::new(
-            FileType::SubnetworkRewards,
+            FileType::RadioRewardShare,
             store_base_path,
-            subnet_rewards_rx,
+            radio_rewards_rx,
         )
         .deposits(Some(file_upload_tx.clone()))
+        .manifest(reward_manifest.clone())
```

Comment:
Do we not want the same 15 roll time on this sink?

Reply:
Fixed